### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by ruzickap**
Improve documentation

### CODE REVIEW

File: docs/kqueen.cli_gcloud_examples.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Create the Google Cloud account and start with `gcloud`:
+
+https://cloud.google.com/pubsub/docs/quickstart-cli
+
+Login to Google Cloud
+
+.. code-block:: bash
+
+   $ cd /tmp/
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: remove $

File: docs/kqueen.cli_gcloud_examples.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
+------------------------------------
+
+Create the Google Cloud account and start with `gcloud`:
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: i prefer to mark each step with number
u may use '#'

File: docs/kqueen.cli_gcloud_examples.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+Create new project in GCE
+
+.. code-block:: bash
+
+   $ GCE_PROJECT_NAME="My Test Project"
+   $ GCE_PROJECT_ID="my-test-project-`date +%F`"
+   $ gcloud projects create "$GCE_PROJECT_ID" --name "$GCE_PROJECT_NAME"
+   Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/my-test-project-2018-05-11].
+   Waiting for [operations/cp.7874714971502871878] to finish...done.
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: remove log output

Comment (naumvd95) [reply docs/kqueen.cli_gcloud_examples.rst]: and $

File: docs/kqueen.cli_gcloud_examples.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: `Google cloud CLI operations with Kqueen`  or smth similar, coz imo this header is unclear

File: docs/kqueen.cli_gcloud_examples.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+https://cloud.google.com/pubsub/docs/quickstart-cli
+
+Login to Google Cloud
+
+.. code-block:: bash
+
+   $ cd /tmp/
+   $ gcloud init
+
+Create new project in GCE
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end

File: docs/kqueen.cli_gcloud_examples.rst [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ GCE_PROJECT_NAME="My Test Project"
+   $ GCE_PROJECT_ID="my-test-project-`date +%F`"
+   $ gcloud projects create "$GCE_PROJECT_ID" --name "$GCE_PROJECT_NAME"
+   Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/my-test-project-2018-05-11].
+   Waiting for [operations/cp.7874714971502871878] to finish...done.
+
+Set new project as default
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end 
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ gcloud config set project $GCE_PROJECT_ID
+
+Enable Billing for the project
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+   $ GCE_ACCOUNT_ID=$(gcloud beta billing accounts list --filter="My Billing Account" --format="value(ACCOUNT_ID)")
+   $ gcloud billing projects link "$GCE_PROJECT_ID" --billing-account="$GCE_ACCOUNT_ID"
+
+Create Service Account and assign it proper rights
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_DISPLAY_NAME="My_Service_Account"
+   $ GCE_SERVICE_ACCOUNT_NAME="my-service-account"
+   $ gcloud iam service-accounts create "$GCE_SERVICE_ACCOUNT_NAME" --display-name "$GCE_SERVICE_ACCOUNT_DISPLAY_NAME"
+
+Generate keys for new service account
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list --filter="$GCE_SERVICE_ACCOUNT_NAME" --format="value(email)")
+   $ gcloud iam service-accounts keys create --iam-account $GCE_SERVICE_ACCOUNT_EMAIL key.json
+
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: describe that user should keep service keys in safe place

File: docs/kqueen.cli_gcloud_examples.rst [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+Assign role to the serviceaccount and bind it to the project
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: spaces and ':'

File: docs/kqueen.cli_gcloud_examples.rst [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ GCE_EMAIL="petr.ruzicka@gmail.com"
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: change email for sane-defaults

File: docs/kqueen.cli_gcloud_examples.rst [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+   $ gcloud iam service-accounts add-iam-policy-binding "$GCE_SERVICE_ACCOUNT_EMAIL" --member="user:$GCE_EMAIL" --role="roles/owner"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/container.clusterAdmin"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/iam.serviceAccountActor"
+
+Start Kqueen (if necessary) and obtain bearer token
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

Comment (naumvd95) [reply docs/kqueen.cli_gcloud_examples.rst]: `if necessary` sounds weird, imo need to refactor this raw

File: docs/kqueen.cli_gcloud_examples.rst [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: '$' signs

File: docs/kqueen.cli_gcloud_examples.rst [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+Create new organization "testorganization" with new user/password "testusername / testpassword":
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data '{ "name": "testorganization", "namespace": "testorganization" }' -X POST 127.0.0.1:5000/api/v1/organizations | jq
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: $

File: docs/kqueen.cli_gcloud_examples.rst [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+   $ ORG_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"  127.0.0.1:5000/api/v1/organizations | jq -r '.[] | select (.name == "testorganization").id')
+   $ echo $ORG_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"username\": \"testusername\", \"organization\": \"Organization:$ORG_ID\", \"role\": \"superadmin\", \"active\": true, \"password\": \"testpassword\" }" -X POST 127.0.0.1:5000/api/v1/users | jq
+
+Switch to new user "testusername" and add new Google Cloud Kubernetes Service provisioner:
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "testusername", "password": "testpassword" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+   $ USER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/users | jq -r '.[] | select (.username == "testusername").id')
+   $ echo $USER_ID
+   $ SERVICE_ACCOUNT_INFO=$(cat ../key.json)
+   $ echo $SERVICE_ACCOUNT_INFO
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testprovisioner\", \"engine\": \"kqueen.engines.GceEngine\", \"owner\": \"User:$USER_ID\", \"parameters\": { \"project\": \"$GCE_PROJECT_ID\", \"service_account_info\": $SERVICE_ACCOUNT_INFO } }" -X POST 127.0.0.1:5000/api/v1/provisioners | jq
+
+Deploy Kubernetes Cluster via kqueen to GCE:
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ...Kqueen with GKE provisioner

File: docs/kqueen.cli_gcloud_examples.rst [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+   $ PROVISIONER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/provisioners | jq -r '.[] | select (.name == "testprovisioner").id')
+   $ echo $PROVISIONER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testcluster\", \"owner\": \"User:$USER_ID\", \"provisioner\": \"Provisioner:$PROVISIONER_ID\", \"metadata\": { \"machine_type\": \"n1-standard-1\", \"node_count\": 1, \"zone\": \"us-central1-a\" } }" -X POST 127.0.0.1:5000/api/v1/clusters | jq
+
+
+---> prubeh curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"  127.0.0.1:5000/api/v1/clusters
+
+
+Download kubeconfig from kqueen "testcluster"
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+Deploy Kubernetes Cluster via kqueen to GCE:
+
+.. code-block:: bash
+
+   $ PROVISIONER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/provisioners | jq -r '.[] | select (.name == "testprovisioner").id')
+   $ echo $PROVISIONER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testcluster\", \"owner\": \"User:$USER_ID\", \"provisioner\": \"Provisioner:$PROVISIONER_ID\", \"metadata\": { \"machine_type\": \"n1-standard-1\", \"node_count\": 1, \"zone\": \"us-central1-a\" } }" -X POST 127.0.0.1:5000/api/v1/clusters | jq
+
+
+---> prubeh curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"  127.0.0.1:5000/api/v1/clusters
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: prubeh?

File: docs/kqueen.cli_gcloud_examples.rst [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+
+Download kubeconfig from kqueen "testcluster"
+
+.. code-block:: bash
+
+   $ CLUSTER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/clusters | jq -r '.[] | select (.name == "testcluster").id')
+   $ echo $CLUSTER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/clusters/$CLUSTER_ID/kubeconfig > kubeconfig.conf
+
+Use kubeconfig and check kubernetes
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 22]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+   $ export KUBECONFIG=$PWD/kubeconfig.conf
+   $ kubectl get nodes
+   $ kubectl get componentstatuses
+   $ kubectl get namespaces
+
+Install helm
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: ':' at the end
mark steps

File: docs/kqueen.cli_gcloud_examples.rst [hunk 23]
```diff
@@ -0,0 +1,137 @@
...
+
```
Comment (naumvd95) [inline docs/kqueen.cli_gcloud_examples.rst]: IMO helm its additional step, describe it

File: docs/kqueen.rst [hunk 24]
```diff
@@ -179,8 +179,8 @@ To set up the NGINX server
 2. Open the docker-compose.production.yml file for editing.
 3. Verify the proxy service configuration. Pay attention on following variables:
 
-  1. ``VHOSTNAME``, the domain name for the KQueen service. Should be the same as the domain name in the generated certificates. By default, the ``NGINX_VHOSTNAME`` from the `.env` file is used. 
-  2. ``SSL_CERTIFICATE_DIR``, the mapped directory for certificates forwarding into a docker container. By default, the ``$NGINX_SSL_CERTIFICATE_DIR/$NGINX_VHOSTNAME`` variable from the `.env` file is used. 
+  1. ``VHOSTNAME``, the domain name for the KQueen service. Should be the same as the domain name in the generated certificates. By default, the ``NGINX_VHOSTNAME`` from the `.env` file is used.
```
Comment (naumvd95) [inline docs/kqueen.rst]: move this changes in separate PR

File: docs/kqueen.rst [hunk 25]
```diff
@@ -409,7 +409,7 @@ To provision a Kubernetes cluster using the Azure Kubernetes Service:
 
 .. note::
```
Comment (naumvd95) [inline docs/kqueen.rst]: move this changes in separate PR

File: docs/kqueen.rst [hunk 26]
```diff
@@ -433,7 +433,7 @@ Backup and recovery ETCD
 
 Etcd is the only stateful component of KQueen. To recover etcd in case of a failure, follow the procedure described in https://coreos.com/etcd/docs/latest/v2/admin_guide.html#disaster-recovery.
 
-.. note:: 
+.. note::
```
Comment (naumvd95) [inline docs/kqueen.rst]: move this changes in separate PR

File: kqueen/blueprints/api/views.py [hunk 27]
```diff
@@ -177,7 +177,7 @@ def cluster_topology_data(pk):
 def cluster_kubeconfig(pk):
     obj = get_object(Cluster, pk, current_identity)
 
-    return jsonify(obj.kubeconfig)
+    return jsonify(obj.get_kubeconfig())
```
Comment (naumvd95) [inline kqueen/blueprints/api/views.py]: why its here??

Comment (naumvd95) [reply kqueen/blueprints/api/views.py]: move this changes in separate PR
and describe main goal of it, does it bugfix?

File: docs/kqueen.cli_gcloud_examples.rst (hunk 1 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: I would like to keep the "$" in front of every command. This is a good practice from other projects (http://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) and it helps to automate running of the commands (byt CI if necessary) and differentiate commands from it's output:

```
$ ls
api.yml  index.rst  kqueen.blueprints.api.rst  kqueen.cli_api_examples.rst     kqueen.engines.rst  kqueen.storages.rst
conf.py  key.json   kqueen.blueprints.rst      kqueen.cli_gcloud_examples.rst  kqueen.rst          Makefile
```

Let me know what you think about it.

File: docs/kqueen.cli_gcloud_examples.rst (hunk 2 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: I don't think it's a good idea. This is not step by step guide and some steps can be mixed with others. This is rarely seen in other documentations...

File: docs/kqueen.cli_gcloud_examples.rst (hunk 3 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: I prefer to keep the outputs wherever is possible - to see real outputs + real examples. 
This helps users to ensure they are getting the right "similar" output (and it should not harm anybody)...

Let me know if you really want to remove all the outputs...

File: docs/kqueen.cli_gcloud_examples.rst (hunk 4 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: Ok... Fixed...

File: docs/kqueen.cli_gcloud_examples.rst (hunk 5 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: Good point...
Fixing everywhere...

File: docs/kqueen.cli_gcloud_examples.rst (hunk 16 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: I'm not going to change it.
According the https://docs.openstack.org/doc-contrib-guide/index.html this is the correct way.

File: docs/kqueen.cli_gcloud_examples.rst [hunk 28]
```diff
@@ -0,0 +1,139 @@
...
+   $ curl -s $(curl -s https://github.com/kubernetes/helm | awk -F \" "/linux-amd64/ { print \$2 }") | tar xvzf - -C /tmp/ linux-amd64/helm
+   $ sudo mv /tmp/linux-amd64/helm /usr/local/bin/
+   $ kubectl create serviceaccount tiller --namespace kube-system
+   $ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+   $ helm init --service-account tiller
+   $ sleep 30
+   $ helm repo update
+
+Install few apps to utilize the cluster:
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: To remove a cluster?

Please add an intro. Smth like:
You can easily remove a cluster using the ... application...

File: docs/kqueen.cli_gcloud_examples.rst [hunk 29]
```diff
@@ -0,0 +1,139 @@
...
+Use kubeconfig and check kubernetes:
+
+.. code-block:: bash
+
+   $ export KUBECONFIG=$PWD/kubeconfig.conf
+   $ kubectl get nodes
+   $ kubectl get componentstatuses
+   $ kubectl get namespaces
+
+Install helm to install application easily:
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: To install Helm:

File: docs/kqueen.cli_gcloud_examples.rst [hunk 30]
```diff
@@ -0,0 +1,139 @@
...
+
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "testusername", "password": "testpassword" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+   $ USER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/users | jq -r '.[] | select (.username == "testusername").id')
+   $ echo $USER_ID
+   $ SERVICE_ACCOUNT_INFO=$(cat ../key.json)
+   $ echo $SERVICE_ACCOUNT_INFO
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testprovisioner\", \"engine\": \"kqueen.engines.GceEngine\", \"owner\": \"User:$USER_ID\", \"parameters\": { \"project\": \"$GCE_PROJECT_ID\", \"service_account_info\": $SERVICE_ACCOUNT_INFO } }" -X POST 127.0.0.1:5000/api/v1/provisioners | jq
+
+Deploy Kubernetes Cluster using Kqueen with GKE provisioner:
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: To deploy a Kubernetes cluster using the GKE provisioner:

File: docs/kqueen.cli_gcloud_examples.rst [hunk 31]
```diff
@@ -0,0 +1,139 @@
...
+
+.. code-block:: bash
+
+   $ GCE_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list --filter="$GCE_SERVICE_ACCOUNT_NAME" --format="value(email)")
+   $ gcloud iam service-accounts keys create --iam-account $GCE_SERVICE_ACCOUNT_EMAIL key.json
+
+Assign role to the serviceaccount and bind it to the project:
+
+.. code-block:: bash
+
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: Please provide a step-by-step procedure (and for other tasks below as well)

File: docs/kqueen.cli_gcloud_examples.rst [hunk 32]
```diff
@@ -0,0 +1,150 @@
...
+For example, you can create a Google Cloud account, create a new Kubernetes cluster (GKE), download the kubeconfig and use it to push application into Kubernetes.
+
+To create / configure the GCE cluster to work with KQueen follow these steps:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Create a Google Cloud account and configure the `gcloud` tool to access your GCE:
+
+https://cloud.google.com/sdk/gcloud/reference/init
+
+#. Login to Google Cloud:
```
Comment (ekhomyakova) [inline docs/kqueen.cli_gcloud_examples.rst]: Log in

File: docs/kqueen.cli_gcloud_examples.rst [hunk 33]
```diff
@@ -0,0 +1,150 @@
...
+.. code-block:: bash
+
+   $ gcloud config set project $GCE_PROJECT_ID
+
+#. Enable Billing for the project:
+
+   $ GCE_ACCOUNT_ID=$(gcloud beta billing accounts list --filter="My Billing Account" --format="value(ACCOUNT_ID)")
+   $ gcloud billing projects link "$GCE_PROJECT_ID" --billing-account="$GCE_ACCOUNT_ID"
+
+#. Create Service Account and assign it proper rights:
```
Comment (ekhomyakova) [inline docs/kqueen.cli_gcloud_examples.rst]: assign proper rights to it?

File: docs/kqueen.cli_gcloud_examples.rst [hunk 34]
```diff
@@ -0,0 +1,150 @@
...
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+#. Create new organization "testorganization" with new user/password "testusername / testpassword":
```
Comment (ekhomyakova) [inline docs/kqueen.cli_gcloud_examples.rst]: Why do you sometimes use "a/b" and sometimes "a / b"? Maybe consistency would be better

File: docs/kqueen.cli_gcloud_examples.rst [hunk 35]
```diff
@@ -0,0 +1,150 @@
...
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data '{ "name": "testorganization", "namespace": "testorganization" }' -X POST 127.0.0.1:5000/api/v1/organizations | jq
```
Comment (ekhomyakova) [inline docs/kqueen.cli_gcloud_examples.rst]: Does it make sense to declare a variable for 127.0.0.1:5000? It would be easier to change it

File: docs/kqueen.cli_gcloud_examples.rst [hunk 36]
```diff
@@ -0,0 +1,150 @@
...
+
+#. Deploy Kubernetes cluster using the GKE provisioner:
+
+.. code-block:: bash
+
+   $ PROVISIONER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/provisioners | jq -r '.[] | select (.name == "testprovisioner").id')
+   $ echo $PROVISIONER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testcluster\", \"owner\": \"User:$USER_ID\", \"provisioner\": \"Provisioner:$PROVISIONER_ID\", \"metadata\": { \"machine_type\": \"n1-standard-1\", \"node_count\": 1, \"zone\": \"us-central1-a\" } }" -X POST 127.0.0.1:5000/api/v1/clusters | jq
+
+#. Check the status of the cluster by query KQueen API (run this command multiple times):
```
Comment (ekhomyakova) [inline docs/kqueen.cli_gcloud_examples.rst]: Maybe it's better to specify the expected output which signifies that status is ok?

File: docs/kqueen.cli_gcloud_examples.rst (hunk 35 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: That's hardcoded in KQueen.
Not sure how to change it easily... It's everywhere written like this (see https://github.com/Mirantis/kqueen/ and search for 5000)

File: docs/kqueen.cli_gcloud_examples.rst (hunk 36 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: I will add the proper output later once I'll do more testing...

File: docs/kqueen.cli_gcloud_examples.rst [hunk 37]
```diff
@@ -0,0 +1,150 @@
...
+#. Assign a role to the service account and bind it to the project:
+
+.. code-block:: bash
+
+   $ GCE_EMAIL="your_email_address@gmail.com"
+   $ gcloud iam service-accounts add-iam-policy-binding "$GCE_SERVICE_ACCOUNT_EMAIL" --member="user:$GCE_EMAIL" --role="roles/owner"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/container.clusterAdmin"
+   $ gcloud projects add-iam-policy-binding $GCE_PROJECT_ID --member="serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL" --role="roles/iam.serviceAccountActor"
+
+To create / configure the GKE cluster using KQueen follow these steps:
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: How does this part differ from line 7?

File: docs/kqueen.cli_gcloud_examples.rst [hunk 38]
```diff
@@ -0,0 +1,150 @@
+Google Cloud CLI operations with KQueen
+---------------------------------------
+
+In KQueen, you can perform a number of Google Cloud operations using the command-line interface (CLI).
+For example, you can create a Google Cloud account, create a new Kubernetes cluster (GKE), download the kubeconfig and use it to push application into Kubernetes.
+
+To create / configure the GCE cluster to work with KQueen follow these steps:
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: Please remove " follow these steps" here and below as well.

File: docs/kqueen.cli_gcloud_examples.rst [hunk 39]
```diff
@@ -0,0 +1,150 @@
+Google Cloud CLI operations with KQueen
+---------------------------------------
+
+In KQueen, you can perform a number of Google Cloud operations using the command-line interface (CLI).
+For example, you can create a Google Cloud account, create a new Kubernetes cluster (GKE), download the kubeconfig and use it to push application into Kubernetes.
```
Comment (mzlatkova) [inline docs/kqueen.cli_gcloud_examples.rst]: an application

File: docs/kqueen.cli_gcloud_examples.rst (hunk 37 above)
Comment (ruzickap) [reply docs/kqueen.cli_gcloud_examples.rst]: fixed
